src/index.js

```diff
@@ -35,12 +35,38 @@ document.querySelector('.addBook').addEventListener('click', (e) => {
 document.querySelector('.books-list').addEventListener('click', (e) => {
   // Eddit task.
   if (e.target.classList.contains('tasks')) {
-    Store.update(e.target);
+    const ide = e.target.parentElement.firstChild.id;
+    e.target.removeAttribute('readonly');
+    e.target.addEventListener('input', () => {
+      const tasks = Store.getTask();
+      // eslint-disable-next-line arrow-body-style
+      const matching = tasks.find((task) => {
+        // eslint-disable-next-line eqeqeq
```

Comment:
- [ ] Going through this file, I found out that you disabled the Linters all over the codes. I would advise you not to disable them, you can solve them manually AND satisfy the general requirements.